hack/run-locally.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash 
+
+# run-locally.sh is some scaffolding around running a local instance of the
+# network operator with the installer.
+# See INSTALLER-HACKING.md
+
+set -o errexit
+set -o nounset
+
+# Install our overrides so the cluster doesn't run the network operator.
+function override() {
+    if [[ ! -e ${CLUSTER_DIR}/manifests/cvo-overrides.yaml ]]; then
+        echo "cannot find cvo-overrides.yaml; please run"
+        echo "openshift-install --dir=${CLUSTER_DIR} create manifests"
+        exit 1
+    fi
+
+    # Patch the CVO to not create the network operator
+    echo "Applying overrides to ${CLUSTER_DIR}/manifests/cvo-overrides.yaml"
+
+    kubectl --kubeconfig=hack/null-kubeconfig patch --type=json --local=true -f=${CLUSTER_DIR}/manifests/cvo-overrides.yaml -p "$(cat hack/overrides-patch.yaml)" -o yaml > ${CLUSTER_DIR}/manifests/.cvo-overrides.yaml.new
+
+    mv ${CLUSTER_DIR}/manifests/.cvo-overrides.yaml.new ${CLUSTER_DIR}/manifests/cvo-overrides.yaml
+}
+
+# Extract the image references from the release image.
+function build_env() {
+    echo "Writing image references to ${CLUSTER_DIR}/env.sh"
+    oc --kubeconfig=hack/null-kubeconfig convert --local=true -f manifests/0000_07_cluster-network-operator_03_daemonset.yaml -ojsonpath='{range .spec.template.spec.containers[0].env[?(@.value)]}{.name}{"="}{.value}{"\n"}' > ${CLUSTER_DIR}/env.sh
```

Comment:
oh. heh. I had assumed you had some reason for extracting the exact version numbers of out of the built image :-)